MapboxSearch/sample/build.gradle

```diff
@@ -15,12 +15,25 @@ if (mapboxApiToken == null) {
     throw new Exception("API token is not specified.")
 }
 
-def enableSBS = true
+def enableSBS = null
+def enableSearchBox = null
 
-if (project.hasProperty('ENABLE_SBS')) {
-    enableSBS = Boolean.valueOf(project.property('ENABLE_SBS'))
-} else if (System.getenv('ENABLE_SBS') != null) {
-    enableSBS = Boolean.valueOf(System.getenv('ENABLE_SBS'))
+if (project.hasProperty('ENABLE_SEARCH_BOX')) {
+    enableSearchBox = Boolean.valueOf(project.property('ENABLE_SEARCH_BOX'))
+} else if (System.getenv('ENABLE_SEARCH_BOX') != null) {
+    enableSearchBox = Boolean.valueOf(System.getenv('ENABLE_SEARCH_BOX'))
+}
+
+if (enableSearchBox != true) {
+    if (project.hasProperty('ENABLE_SBS')) {
+        enableSBS = Boolean.valueOf(project.property('ENABLE_SBS'))
+    } else if (System.getenv('ENABLE_SBS') != null) {
+        enableSBS = Boolean.valueOf(System.getenv('ENABLE_SBS'))
+    }
+
+    enableSearchBox = false
+} else {
+    enableSBS = false
```

Comment:
Should we just change this to a constant of type "SDS", "GEOCODING", "SEARCH_BOX"?

Reply:
It seems like this may be an outdated concept when we were still in development for the SBS API ("Enable_SBS"), these days SBS/Search Box is available for everyone. Moving away from that naming convention seems like it could be helpful in reducing questions developers might ask. (I think this is in line with your comment Mike)

Reply:
I refactored this part, please take a look and comment on the new version.